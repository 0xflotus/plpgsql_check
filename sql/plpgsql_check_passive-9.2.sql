load 'plpgsql';
load 'plpgsql_check';

set plpgsql_check.mode = 'every_start';

create or replace function f1()
returns void as $$
begin
  if false then
    raise notice '%', 1, 2;
  end if;
end;
$$ language plpgsql;

select f1();

drop function f1();

create or replace function f1()
returns void as $$
begin
  if false then
    raise notice '% %';
  end if;
end;
$$ language plpgsql;

select f1();

drop function f1();
