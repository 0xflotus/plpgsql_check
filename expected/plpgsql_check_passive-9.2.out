load 'plpgsql';
load 'plpgsql_check';
set plpgsql_check.mode = 'every_start';
create or replace function f1()
returns void as $$
begin
  if false then
    raise notice '%', 1, 2;
  end if;
end;
$$ language plpgsql;
select f1();
ERROR:  too many parameters specified for RAISE
CONTEXT:  PL/pgSQL function f1() line 4 at RAISE
drop function f1();
create or replace function f1()
returns void as $$
begin
  if false then
    raise notice '% %';
  end if;
end;
$$ language plpgsql;
select f1();
ERROR:  too few parameters specified for RAISE
CONTEXT:  PL/pgSQL function f1() line 4 at RAISE
drop function f1();
